doc/userDocs/usersGuide.md

```diff
@@ -141,7 +140,32 @@ The image below shows the lines that need to be edited in the configuration file
 2. Update *kubeconfigFile* to the full path to your kubeconfig file.  The paths will be identical if using a single cluster for both the app and driver.
 3. Update *kubeconfigContext* parameters with the name of the context for your cluster from the kubeconfig file, such as `kubernetes-admin@kubernetes`.
     - Set an empty string (`""`) to use the current-context defined in the kubeconfigFile.
-4. Set *useLoadBalancer* to `true` if your cluster supports LoadBalancer services. Otherwise, set it to `false` to use a NodePort for ingress.
+4. Choose a value for *appIngressMethod* that reflects the method you want to use
+  for communication between the workload drivers and the applications.
+    - If the workload drivers and applications are running on different Kubernetes 
+      clusters, you can choose either:
+        - `loadbalancer`:  Traffic from the workload drivers to the applications will
+          use the external IP address of a Kubernetes LoadBalancer service. Your cluster must 
+          support provisioning external IP addresses for LoadBalancer services. 
+        - `nodeport`:  Traffic from the workload drivers to the applications will
+          use the external IP addresses of the Kubernetes Nodes via a NodePort service. Your cluster must 
+          support exposing the node IP addresses externally using NodePort services.
+    - If the workload drivers and the applications are running on the same Kubernetes 
+      cluster, you can choose among:
+        - `none`: Traffic from the workload drivers to the applications will use a 
```

Comment:
Should 'none' be 'clusterip' ?

Reply:
yes.  Good catch.  I initially had a slightly different interpretation of the appIngressMethod parameter, and was using _none_ rather than _clusterip_.